app.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const path = require('path');
+
+const compression = require('compression');
+
+const express = require('express');
+
+const st = require('st');
+
+
+function createServer(config) {
+
+  const {
+    env,
+    port,
+    hostname
+  } = config;
+
+  const app = express();
+
+  app.set('port', port);
+  app.set('env', env);
+  app.set('hostname', hostname);
+
+  if (env === 'development') {
+    app.use(loggingMiddleware());
+  }
+
+  app.use(compression());
+
+  if (process.env.FORCE_HTTPS) {
```

Comment:
I would suggest to make FORCE_HTTPS the default (so to optimize for search-engine ranking rather than having 100% compatability)

Reply:
Not sure what you mean. In the heroku apps, FORCE_HTTPS is set so the app redirects to https. However, in development, you don't necessarily need to set up SSL for localhost when you only review changes.

Reply:
Ah okay, I did not know that FORCE_HTTPS is set in the heroku apps.